sqlmesh/core/audit/builtin.py

```diff
@@ -7,22 +7,26 @@
 # not_null(columns=[column_1, column_2])
 not_null_audit = ModelAudit(
     name="not_null",
+    defaults={"condition": exp.true()},
```

Comment:
I can't recall whether plain `True` universally works across all engines. I remember having to do `1 = 1` expression instead. Or maybe it's only relevant when there is just one condition in the `WHERE` clause. @tobymao @eakmanrq @treysp any chance one of you knows about what I'm referring to?

Reply:
why do you need any condition at all? sqlglot will remove empty where statements

Reply:
i think TRUE doesn't work in tsql, only 1 =1, but i think we may have a sqlglot transpiler for that

Reply:
sqlglot transpiles to 1=1

```
>>> exp.true().sql(dialect="tsql")
'(1 = 1)'
```

Reply:
@tobymao what do you mean, how would a user be able to _optionally_ supply a condition?
If they dont provide one, we need a default in the context so there is no KeyError when trying to render the expression?
If the user does provide a condition, we need to specify where it goes (in the where clause with @condition)

Can you share and example of what you mean?

Reply:
i guess we need a new macro def and_ or def or_ that will take in optionals and construct the proper query, can chat with you over slack if you need more info

sqlglot has helpers

exp.and_ that does this i think